docs/delayed_embed.md

```diff
@@ -0,0 +1,116 @@
+# Delayed embed
+
+You may decide to combine pgai with [Timescaledb Background Actions][tsdb_actions].
+and you can also use the `pgai` extension to run background actions.
+
+## Security concerns
+
+There are some security concerns when running background actions, specially
+because you may need to access some external service, like OpenAI API.
+
+You should be aware of how you're storing your keys and how it can be accessed.
+
+The background action should be run by a user that has the necessary permissions
+but it also needs to access some shared memory with the api key or receive it
+via payload.
+
+If you store it in the payload, make sure you're using a secure connection and
+also that your secrets are renewed frequently.
+
+You can also setup this settings at your server level, for example, you can
+add a new config to the system:
+
+```bash
+echo "ai.openai_api_key=${API_KEY}" >> /path/to/postgresql.conf
```

Comment:
still not sure if this is a valid option.

Reply:
I think maybe we can go ahead and merge this if we add a big warning/disclaimer here that putting the API key in the conf file means

1. your API key is in a plain text file
2. any database user can `show ai.openai_api_key` and see it

"use your own judgment" and "proceed at your own risk"